middleware/service/source/manager/handle.cpp

```diff
@@ -278,7 +290,7 @@ namespace casual
                            log::error( code::casual::invalid_semantics, " callee terminated with pending reply to caller - callee: ", 
                                  event.state.pid, " - caller: ", caller.process.pid);
 
-                           local::error::reply( state, caller, common::code::xatmi::service_error);
+                           local::error::reply( state, caller, common::code::xatmi::service_error, common::process::Handle{ event.state.pid});
```

Comment:
I think `state.remove(...)` should return what's needed to send error reply.
A vector of _removed-wrapper_. that holds `Caller` and `process::Handle` (callee). If anybody knows this, it's `state.remove( ...)`, after the call, the related state to the pid is gone.